cli/client/authentication.go

```diff
@@ -74,12 +76,43 @@ func (client *RestClient) Logout(token string) error {
 }
 
 // RefreshAccessToken returns a new access token given valid refresh token
-func (client *RestClient) RefreshAccessToken(token string) (*types.Tokens, error) {
-	res, err := client.R().
-		SetHeader("Content-Type", "application/json").
-		SetBody(map[string]string{"refresh_token": token}).
-		SetResult(&types.Tokens{}).
-		Post("/auth/token")
+func (client *RestClient) RefreshAccessToken(tokens *corev2.Tokens) (*corev2.Tokens, error) {
+	var err error
+	var res *resty.Response
+	url := "/auth/token"
+
+	// Parse the access claims so we can determine the issuer URL
+	parser := new(jwt.Parser)
+	claims := &corev2.Claims{}
+	_, _, _ = parser.ParseUnverified(tokens.Access, claims)
```

Comment:
Because the client (sensuctl) doesn't necessarily have access to the secret key or the public key to validate the JWT; it will still be validated on the server side though